CivicActions/accessibility
# Cypress update and also a "tab" test

Closes #649

I updated Cypress and axe-core which flagged a11y errors in the accordions so I updated the logic to use a button in the header. Adjusted the styling to keep it the same as before. Here is the error:

```
Issue Description
Ensures ARIA attributes are allowed for an element's role

Element location
.accordion:nth-child(1) > .accordion-heading[data-toggle="1"]

Element source
<div tabindex="0" class="accordion-heading" data-toggle="1" aria-expanded="true">
                                <h2>Personas</h2>
                            </div>

To solve this issue, you need to...
Fix the following:
ARIA attribute is not allowed: aria-expanded="true"
```

Axe rule: https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr?application=AxeChrome

Also tried to get https://www.npmjs.com/package/cypress-plugin-tab working but it didn't support 'enter' after tabbing so added a simple test that uses 'enter' keypress event, not perfect but something. Will have to add one for the accordion as well.

**QA steps**
* Load https://accessibility-qa.civicactions.com/roles/procurement.
* Test the accordions and confirm they work as expected.
* Click the 'Expand All' button.
* Confirm that it expands all accordions and changes to 'Collapse All'.
* Click the 'Collapse All' button.
* Confirm that it collapses all accordions and changes to 'Expand All'.
* Try it with some accordions expanded and collapsed and confirm the expand/collapse still behaves as expected.
